### PR TITLE
Fix/silence remaining warnings

### PR DESCRIPTION
**Pull request by Rubonnek**
Fixes/silence the remaining warnings if enabled in Project Settings.

### CODE REVIEW

File: addons/resource_remaps/resource_remaps_plugin.gd [hunk 1]
```diff
@@ -23,7 +23,6 @@
 @tool
 extends EditorPlugin
 
-const CONTROL_CONTAINER = CONTAINER_PROJECT_SETTING_TAB_RIGHT
```
Comment (allenwp) [inline addons/resource_remaps/resource_remaps_plugin.gd:26]: I'd prefer this approach instead:

`const CONTROL_CONTAINER: CustomControlContainer = CONTAINER_PROJECT_SETTING_TAB_RIGHT`

File: addons/resource_remaps/resource_remaps_control.gd.uid
Comment (allenwp) [inline addons/resource_remaps/resource_remaps_control.gd.uid:1]: Oh, I didn't see these at first. I remember the `.gd.uid` PR was removed from `master` before any stable releases... I'm not sure how you generated these and I would like to not add them until a stable release of Godot adds them...

Comment (allenwp) [reply addons/resource_remaps/resource_remaps_control.gd.uid:1]: Ah, I see that godotengine/godot#97352 was just merged! This is the second time that this sort of thing has been merged and the previous time it was reverted before it was released into a stable build.

I'd like to not include these `.uid` files until the stable release. (I won't forget about this plugin and I will make sure to update it very soon after `uid` files go live in a stable Godot version!)

File: addons/resource_remaps/resource_remaps_plugin.gd (hunk 1 above)
Comment (Rubonnek) [reply addons/resource_remaps/resource_remaps_plugin.gd:26]: I've updated the PR accordingly.

**Comment by allenwp** [comment]
Thanks @Rubonnek!

**Pull request merged**